recipes/hdf5/all/conanfile.py

```diff
@@ -46,6 +46,12 @@ class Hdf5Conan(ConanFile):
         "enable_unsupported": False
     }
 
+    def get_crossbuild_emulator(self):
+        for (k, v) in self.conf.items():
+            if k == "tools.qemu:path":
```

Comment:
Where does this conf name come from? There's nothing like it in the client - And it won't let recipes create new `tool.` confs not built-in.

I'm guessing you are looking for something like `self.conf.get("user.qemu:path")` or similar in this case :)

Reply:
As was unable to find an official qemu configuration setting. Therefore, I decided to define my own name. This was the best thing I could think off at the time🤷‍♂️.
But you preferer, I can certainly switch to `self.conf.get("user.qemu:path")`.

Reply:
@RubenRBS Should I go back to my iterating over conf items approach. Or is there another API I am missing?